test/mitmproxy/utils/test_human.py

```diff
@@ -46,3 +46,8 @@ def test_pretty_duration():
     assert human.pretty_duration(10000) == "10000s"
     assert human.pretty_duration(1.123) == "1.12s"
     assert human.pretty_duration(0.123) == "123ms"
+
+
+def test_format_address():
+    assert human.format_address(('::1','55700','0','0')) == "[::1]:55700"
+    assert human.format_address(('::ffff:127.0.0.1','55700','0','0')) == "127.0.0.1:55700"
```

Comment:
As I mentioned a couple of times now, this function will also be used in other contexts (see e.g. https://github.com/mitmproxy/mitmproxy/pull/2183#pullrequestreview-27747348) where it can be a plain IPv4 address or a hostname.